src/test/java/com/aws/greengrass/lifecyclemanager/LogManagerHelperTest.java

```diff
@@ -123,9 +131,122 @@ void GIVEN_mock_service_WHEN_getComponentLogger_THEN_logs_to_correct_log_file()
         assertEquals(0, ggLogFile.length());
     }
 
+    @Test
+    void GIVEN_mock_service_logger_WHEN_reconfigure_THEN_change_applied_correctly() throws IOException {
```

Comment:
This added test is great and is going be the guard for the logger config change code. If this gets long, I may suggest split it into multiple tests for various re-configurable field.